sampleapp/src/main/java/com/zendesk/sampleapp/MainActivity.kt

```diff
@@ -0,0 +1,75 @@
+package com.zendesk.sampleapp
+
+import android.app.NotificationChannel
+import android.app.NotificationManager
+import android.content.Context
+import android.support.v7.app.AppCompatActivity
+import android.os.Bundle
+import android.util.Log
+import io.outbound.sdk.Event
+import io.outbound.sdk.Outbound
+import io.outbound.sdk.User
+import kotlinx.android.synthetic.main.activity_main.*
+
+const val LOG_TAG = "SampleMainActivity"
+
+const val NOTIFICATION_CHANNEL_ID = "outbound_sample_app_notifications"
+const val NOTIFICATION_CHANNEL_NAME = "Outbound Sample App Notifications"
+
+//TODO: Replace with your Connect private api key
+const val CONNECT_API_KEY: String = "xxx"
```

Comment:
Again it would be nice to add something a bit more adaptive/user-friendly like what we have in Core and Support (and possibly AnswerBot... I'm not sure), but that's something for another day. 
Not sure if you want the TODOs. Android Studio will likely complain at you about them all the time.